### DISCUSSION TITLE
Can Qfield create new point datasets or just edit existing ones

### DISCUSSION THREAD

**Discussion by kaddison9768** [opengisch/QField#4987] [Q&A]
Is there any way QField can make a new feature class (shapefile, or whatever format) in program? Or can it only load pre-existing ones? My use case is extremely ad-hoc so I'd like the ability to create new FCs in program instead of having to plan it out more.

**Comment by suricactus** [comment]
You cannot create layers within QField. However, you can prepare a GeoPackage with some int, float, boolean and text fields and keep it on your device. Whenever you need to do ad-hoc data collection, you can import a copy of it in QField and start your digitizing.
